### ISSUE TITLE
manifest.json not created or found

### ISSUE THREAD

**Issue by aaronbannin** [apache/incubator-superset#3255]
Make sure these boxes are checked before submitting your issue - thank you!

- [x ] I have checked the superset logs for python stacktraces and included it here as text if any
- [x ] I have reproduced the issue with at least the latest released version of superset
- [x ] I have checked the issue tracker for the same issue and I haven't found one similar

### Superset version
`Superset 0.19.0`

### Expected results
Accessing `localhost:8088` shows home page.

### Actual results
Accessing `localhost:8088` shows a partially loaded homepage. Selecting a link shows a functional page.

Server logs:
```
Starting server with command: 
gunicorn -w 2 --timeout 60 -b  0.0.0.0:8088 --limit-request-line 0 --limit-request-field_size 0 superset:app

[2017-08-07 21:03:03 -0700] [59190] [INFO] Starting gunicorn 19.7.1
[2017-08-07 21:03:03 -0700] [59190] [INFO] Listening at: http://0.0.0.0:8088 (59190)
[2017-08-07 21:03:03 -0700] [59190] [INFO] Using worker: sync
[2017-08-07 21:03:03 -0700] [59193] [INFO] Booting worker with pid: 59193
[2017-08-07 21:03:03 -0700] [59194] [INFO] Booting worker with pid: 59194
no manifest file found at /usr/local/projects/incubator-superset/superset/static/assets/dist/manifest.json
```

### Steps to reproduce
Followed installation instructions here: `https://superset.incubator.apache.org/installation.html`

Using Python 3 with venv.
<img width="1226" alt="screen shot 2017-08-07 at 9 15 02 pm" src="https://user-images.githubusercontent.com/6948336/29056106-8ead49e2-7bb5-11e7-95dd-c1dfdd9663ac.png">

**Comment by mistercrunch** [comment]
looking at the message `no manifest file found at /usr/local/projects/incubator-superset/superset/static/assets/dist/manifest.json` it doesn't seem like you are using as venv as you claim. I'd expect to see `site-packages` somewhere in there

Maybe leftover from a previous install?

**Comment by aaronbannin** [comment]
With the command line:
```
(venv) abannin@macbook-pro:/usr/local/projects/incubator-superset$ 
(venv) abannin@macbook-pro:/usr/local/projects/incubator-superset$ superset runserver
Starting server with command: 
gunicorn -w 2 --timeout 60 -b  0.0.0.0:8088 --limit-request-line 0 --limit-request-field_size 0 superset:app

[2017-08-07 22:44:27 -0700] [60162] [INFO] Starting gunicorn 19.7.1
[2017-08-07 22:44:27 -0700] [60162] [INFO] Listening at: http://0.0.0.0:8088 (60162)
[2017-08-07 22:44:27 -0700] [60162] [INFO] Using worker: sync
[2017-08-07 22:44:27 -0700] [60165] [INFO] Booting worker with pid: 60165
[2017-08-07 22:44:27 -0700] [60166] [INFO] Booting worker with pid: 60166
no manifest file found at /usr/local/projects/incubator-superset/superset/static/assets/dist/manifest.json
```

`ls -ltr` from the `venv` dir:
```
(venv) abannin@macbook-pro:/usr/local/projects/incubator-superset/venv$ ll
total 16
drwxr-xr-x   3 abannin  admin   102B Aug  7 21:08 lib
drwxr-xr-x   3 abannin  admin   102B Aug  7 21:08 include
lrwxr-xr-x   1 abannin  admin    79B Aug  7 21:08 .Python -> /usr/local/Cellar/python3/3.6.2/Frameworks/Python.framework/Versions/3.6/Python
drwxr-xr-x  34 abannin  admin   1.1K Aug  7 21:08 ..
-rw-r--r--   1 abannin  admin    60B Aug  7 21:08 pip-selfcheck.json
drwxr-xr-x   7 abannin  admin   238B Aug  7 21:08 .
drwxr-xr-x  49 abannin  admin   1.6K Aug  7 21:09 bin
```

`ls -ltr` of `static/assets/`, no `dist` dir. 
```
(venv) abannin@macbook-pro:/usr/local/projects/incubator-superset/superset/static/assets$ ll
total 176
-rw-r--r--   1 abannin  admin   3.6K Aug  5 15:31 webpack.config.js
drwxr-xr-x  46 abannin  admin   1.5K Aug  5 15:31 visualizations
drwxr-xr-x   4 abannin  admin   136B Aug  5 15:31 vendor
drwxr-xr-x   4 abannin  admin   136B Aug  5 15:31 utils
drwxr-xr-x  13 abannin  admin   442B Aug  5 15:31 stylesheets
drwxr-xr-x   4 abannin  admin   136B Aug  5 15:31 spec
-rw-r--r--   1 abannin  admin   4.3K Aug  5 15:31 package.json
-rwxr-xr-x   1 abannin  admin   326B Aug  5 15:31 js_build.sh
drwxr-xr-x  15 abannin  admin   510B Aug  5 15:31 javascripts
drwxr-xr-x  17 abannin  admin   578B Aug  5 15:31 images
lrwxr-xr-x   1 abannin  admin    23B Aug  5 15:31 docs -> ../../docs/_build/html/
drwxr-xr-x  10 abannin  admin   340B Aug  5 15:31 branding
-rw-r--r--   1 abannin  admin    48K Aug  5 15:31 backendSync.json
-rw-r--r--   1 abannin  admin   1.3K Aug  5 15:31 .istanbul.yml
-rw-r--r--   1 abannin  admin   1.0K Aug  5 15:31 .eslintrc
-rw-r--r--   1 abannin  admin   113B Aug  5 15:31 .eslintignore
-rw-r--r--   1 abannin  admin    49B Aug  5 15:31 .babelrc
drwxr-xr-x  19 abannin  admin   646B Aug  5 15:31 .
drwxr-xr-x  31 abannin  admin   1.0K Aug  6 22:55 ..
```

I've tried uninstalling & reinstalling with `pip` and rebuilding the `venv`. Can't find any hints as to what would generate the required assets.

**Comment by aaronbannin** [comment]
The `site-packages` from `venv`:

```
(venv) abannin@macbook-pro:/usr/local/projects/incubator-superset/venv$ cd lib/python3.6/site-packages/
(venv) abannin@macbook-pro:/usr/local/projects/incubator-superset/venv/lib/python3.6/site-packages$ ll
total 952
drwxr-xr-x   58 abannin  admin   1.9K Aug  7 21:08 ..
-rw-r--r--    1 abannin  admin   126B Aug  7 21:08 easy_install.py
drwxr-xr-x   12 abannin  admin   408B Aug  7 21:08 setuptools-36.2.7.dist-info
drwxr-xr-x   38 abannin  admin   1.3K Aug  7 21:08 setuptools
drwxr-xr-x    7 abannin  admin   238B Aug  7 21:08 pkg_resources
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:08 wheel-0.29.0.dist-info
drwxr-xr-x   20 abannin  admin   680B Aug  7 21:08 wheel
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:08 pip-9.0.1.dist-info
drwxr-xr-x   23 abannin  admin   782B Aug  7 21:08 pip
-rw-r--r--    1 abannin  admin    29K Aug  7 21:09 six.py
-rwxr-xr-x    1 abannin  admin   345K Aug  7 21:09 _cffi_backend.cpython-36m-darwin.so
-rw-r--r--    1 abannin  admin    22K Aug  7 21:09 flask_openid.py
-rw-r--r--    1 abannin  admin    30K Aug  7 21:09 flask_login.py
-rw-r--r--    1 abannin  admin    31K Aug  7 21:09 itsdangerous.py
-rw-r--r--    1 abannin  admin   2.7K Aug  7 21:09 editor.py
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 thriftpy-0.3.9.dist-info
drwxr-xr-x   15 abannin  admin   510B Aug  7 21:09 thriftpy
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 thrift_sasl-0.2.1.dist-info
drwxr-xr-x    5 abannin  admin   170B Aug  7 21:09 thrift_sasl
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 six-1.10.0.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 sasl-0.2.1.dist-info
drwxr-xr-x    8 abannin  admin   272B Aug  7 21:09 sasl
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 ply-3.10.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 ply
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 markupsafe
drwxr-xr-x   29 abannin  admin   986B Aug  7 21:09 jinja2
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 MarkupSafe-1.0.dist-info
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:09 Jinja2-2.9.6.dist-info
drwxr-xr-x   27 abannin  admin   918B Aug  7 21:09 werkzeug
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 python_dateutil-2.6.0.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 pycparser-2.18.dist-info
drwxr-xr-x   16 abannin  admin   544B Aug  7 21:09 pycparser
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 itsdangerous-0.24.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 idna-2.6.dist-info
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:09 idna
drwxr-xr-x    6 abannin  admin   204B Aug  7 21:09 flask_testing
drwxr-xr-x   24 abannin  admin   816B Aug  7 21:09 flask
drwxr-xr-x   12 abannin  admin   408B Aug  7 21:09 dateutil
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 click-6.7.dist-info
drwxr-xr-x   20 abannin  admin   680B Aug  7 21:09 click
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 cffi-1.10.0.dist-info
drwxr-xr-x   21 abannin  admin   714B Aug  7 21:09 cffi
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 asn1crypto-0.22.0.dist-info
drwxr-xr-x   29 abannin  admin   986B Aug  7 21:09 asn1crypto
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 Werkzeug-0.12.2.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 Flask_Testing-0.6.2.dist-info
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:09 Flask-0.12.2.dist-info
drwxr-xr-x    7 abannin  admin   238B Aug  7 21:09 libfuturize
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:09 gunicorn-19.7.1.dist-info
drwxr-xr-x   22 abannin  admin   748B Aug  7 21:09 gunicorn
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:09 future
drwxr-xr-x    7 abannin  admin   238B Aug  7 21:09 flask_cache
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 cryptography-1.9.dist-info
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 cryptography
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 colorama-0.3.9.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 colorama
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 Flask_Cache-0.13.1.dist-info
drwxr-xr-x   26 abannin  admin   884B Aug  7 21:09 sqlalchemy
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 python_editor-1.0.3.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 past
drwxr-xr-x    6 abannin  admin   204B Aug  7 21:09 libpasteurize
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 future-0.16.0.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 SQLAlchemy-1.1.9.dist-info
drwxr-xr-x   21 abannin  admin   714B Aug  7 21:09 mako
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 jmespath-0.9.3.dist-info
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:09 jmespath
drwxr-xr-x    5 abannin  admin   170B Aug  7 21:09 flask_sqlalchemy
drwxr-xr-x    7 abannin  admin   238B Aug  7 21:09 flask_script
drwxr-xr-x    6 abannin  admin   204B Aug  7 21:09 flask_migrate
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 alembic-0.9.4.dist-info
drwxr-xr-x   16 abannin  admin   544B Aug  7 21:09 alembic
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 Mako-1.0.7.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 Flask_Script-2.0.5.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 Flask_SQLAlchemy-2.1.dist-info
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 Flask_Migrate-2.0.3.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 docutils-0.14.dist-info
drwxr-xr-x   17 abannin  admin   578B Aug  7 21:09 docutils
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 simplejson-3.10.0.dist-info
drwxr-xr-x   12 abannin  admin   408B Aug  7 21:09 simplejson
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 s3transfer-0.1.10.dist-info
drwxr-xr-x   15 abannin  admin   510B Aug  7 21:09 s3transfer
drwxr-xr-x   43 abannin  admin   1.4K Aug  7 21:09 chardet
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 botocore-1.5.92.dist-info
drwxr-xr-x   35 abannin  admin   1.2K Aug  7 21:09 botocore
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 boto3-1.4.4.dist-info
drwxr-xr-x   15 abannin  admin   510B Aug  7 21:09 boto3
drwxr-xr-x   15 abannin  admin   510B Aug  7 21:09 wtforms
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 urllib3-1.21.1.dist-info
drwxr-xr-x   16 abannin  admin   544B Aug  7 21:09 urllib3
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 thrift-0.10.0.dist-info
drwxr-xr-x   13 abannin  admin   442B Aug  7 21:09 thrift
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 requests-2.17.3.dist-info
drwxr-xr-x   21 abannin  admin   714B Aug  7 21:09 requests
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:09 pyhive
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 pydruid-0.3.1.dist-info
drwxr-xr-x    8 abannin  admin   272B Aug  7 21:09 pydruid
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 parsedatetime-2.0.dist-info
drwxr-xr-x    8 abannin  admin   272B Aug  7 21:09 parsedatetime
drwxr-xr-x   11 abannin  admin   374B Aug  7 21:09 flask_wtf
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 chardet-3.0.4.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 certifi-2017.7.27.1.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 certifi
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 WTForms-2.1.dist-info
drwxr-xr-x    7 abannin  admin   238B Aug  7 21:09 TCLIService
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 PyHive-0.4.0.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 tornado-4.2.dist-info
drwxr-xr-x   36 abannin  admin   1.2K Aug  7 21:09 tornado
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 pytz-2017.2.dist-info
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 pytz
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 Flask_WTF-0.14.2.dist-info
drwxr-xr-x    5 abannin  admin   170B Aug  7 21:09 funtests
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 billiard-3.3.0.23.dist-info
drwxr-xr-x   25 abannin  admin   850B Aug  7 21:09 billiard
drwxr-xr-x   19 abannin  admin   646B Aug  7 21:09 babel
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 anyjson-0.3.3.dist-info
drwxr-xr-x    4 abannin  admin   136B Aug  7 21:09 anyjson
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 amqp-1.4.9.dist-info
drwxr-xr-x   16 abannin  admin   544B Aug  7 21:09 amqp
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 Babel-2.4.0.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 kombu-3.0.37.dist-info
drwxr-xr-x   26 abannin  admin   884B Aug  7 21:09 kombu
drwxr-xr-x   22 abannin  admin   748B Aug  7 21:09 sqlalchemy_utils
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 python3_openid-3.1.0.dist-info
drwxr-xr-x   20 abannin  admin   680B Aug  7 21:09 openid
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 humanize-0.5.1.dist-info
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 humanize
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 flower-0.9.1.dist-info
drwxr-xr-x   15 abannin  admin   510B Aug  7 21:09 flower
drwxr-xr-x    6 abannin  admin   204B Aug  7 21:09 flask_babel
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 defusedxml-0.5.0.dist-info
drwxr-xr-x   14 abannin  admin   476B Aug  7 21:09 defusedxml
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 celery-3.1.25.dist-info
drwxr-xr-x   32 abannin  admin   1.1K Aug  7 21:09 celery
drwxr-xr-x    8 abannin  admin   272B Aug  7 21:09 __pycache__
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 SQLAlchemy_Utils-0.32.14.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 Flask_OpenID-1.2.5.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 Flask_Login-0.2.11.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 Flask_Babel-0.11.2.dist-info
drwxr-xr-x   16 abannin  admin   544B Aug  7 21:09 markdown
drwxr-xr-x   33 abannin  admin   1.1K Aug  7 21:09 flask_appbuilder
drwxr-xr-x    4 abannin  admin   136B Aug  7 21:09 examples
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 Markdown-2.6.8.dist-info
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 Flask_AppBuilder-1.9.1.dist-info
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 numpy-1.13.1.dist-info
drwxr-xr-x   28 abannin  admin   952B Aug  7 21:09 numpy
drwxr-xr-x   10 abannin  admin   340B Aug  7 21:09 sqlparse-0.2.3.dist-info
drwxr-xr-x   16 abannin  admin   544B Aug  7 21:09 sqlparse
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 pandas-0.20.2.dist-info
drwxr-xr-x   26 abannin  admin   884B Aug  7 21:09 pandas
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:09 superset-0.19.0.dist-info
drwxr-xr-x   29 abannin  admin   986B Aug  7 21:10 superset
drwxr-xr-x    9 abannin  admin   306B Aug  7 21:27 psycopg2-2.7.3.dist-info
drwxr-xr-x   17 abannin  admin   578B Aug  7 21:27 psycopg2
drwxr-xr-x  154 abannin  admin   5.1K Aug  7 21:27 .
```

**Comment by mistercrunch** [comment]
did you ever run `npm run dev`?

**Comment by aaronbannin** [comment]
This is fixed by uninstalling then reinstalling Node. I can't see any version difference, so maybe I just had a corrupted install or expired build?

I did not run `npm run dev`, as it's not included here: https://superset.incubator.apache.org/installation.html

I went into `~/superset/static/assets` and ran `npm run dev` and it failed. After a bit of digging, I decided to try `brew uninstall node` and then `brew install node`. Everything is working now, thanks for the pointer!
